### PR TITLE
Add multipart content types to HttpContentType

### PR DESCRIPTION
**Pull request by Leo506**

### CODE REVIEW

File: src/Arbus.Network/HttpContentType.cs [hunk 1]
```diff
@@ -17,4 +16,6 @@ public static class Text
     }
 
     public const string ImageSvgXmlContentType = "image/svg+xml";
+    
+    public const string MultipartFormData = "multipart/form-data";
```
Comment (evgenyvalavin) [inline src/Arbus.Network/HttpContentType.cs]: I believe we better separate class for multipart content-type as we did with Application.cs above.
Multipart content-type has four different values: [All possible values of HTTP Content-type header](https://www.geeksforgeeks.org/http-headers-content-type/)

**Review by evgenyvalavin — CHANGES_REQUESTED** [review]

File: src/Arbus.Network/HttpContentType.cs (hunk 1 above)
Comment (Leo506) [reply src/Arbus.Network/HttpContentType.cs]: Should I add all 4 multipart content types or we will be sufficient by this one?

Comment (evgenyvalavin) [reply src/Arbus.Network/HttpContentType.cs]: That's a great idea! As Arbus.Network is a public package and anyone might use it for any purpose

**Review by evgenyvalavin — APPROVED** [review]

**Comment by evgenyvalavin** [comment]
LGTM

**Pull request merged**